app/helpers/templates_helper.rb

```diff
@@ -53,4 +53,31 @@ def template_input_types_options(keys = TemplateInput::TYPES.keys)
     keys.map!(&:to_s)
     TemplateInput::TYPES.select { |k, _| keys.include?(k.to_s) }.map { |key, name| [ _(name), key ] }
   end
+
+  def hide_class(obj)
```

Comment:
this is very generic name and the helpers are included in all views, so it could cause problems elsewhere...
Could you give it more specific name, please?